File: 782-jewels-and-stones/jewels-and-stones.js
var numJewelsInStones = function(jewels, stones) {
    let hash = {};
    let count = 0;
    for (let char of jewels){
        hash[char] = true;
    }
    for (let char of stones){
        if (hash[char])
        count++;
    }
    return count;
};

File: 782-jewels-and-stones/jewels-and-stones.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        hash = {}
        count = 0
        for each in jewels:
            hash[each] = True
        for each in stones:
            if each in hash:
                count+=1
        return count
        